docs/api/covidcast-signals/covid-trends-and-impact-survey.md

```diff
@@ -11,8 +11,8 @@ permalink: api/covidcast-signals/fb-survey.html
 
 * **Source name:** `fb-survey`
 * **Earliest issue available:** April 29, 2020
-* **Number of data revisions since May 19, 2020:** 1
-* **Date of last change:** [June 3, 2020](../covidcast_changelog.md#fb-survey)
+* **Number of data revisions since 19 May, 2020:** 1 (see [data revision docs](../covidcast_changelog.md#fb-survey))
```

Comment:
I would suggest leaving this date's format as-is to keep it consistent with the other dates on the page.  You could change the other dates to match, but that seems like more work than its worth.  (This also applies to ~all the other instances of this line in the other files)
```suggestion
* **Number of data revisions since May 19, 2020:** 1 (see [data revision docs](../covidcast_changelog.md#fb-survey))
```